nixos/modules/services/misc/paperless.nix

```
@@ -571,6 +585,17 @@ in
             gid = config.ids.gids.paperless;
           };
         };
+
+        services.gotenberg = lib.mkIf cfg.configureTika {
+          enable = true;
+          chromium.disableJavascript = true;
+          extraArgs = [ "--chromium-allow-list=file:///tmp/.*" ];
```

```suggestion
					# https://github.com/paperless-ngx/paperless-ngx/blob/278ef3a364c4b8c0c914160756bfee1a6986e662/docker/compose/docker-compose.sqlite-tika.yml#L64-L69
          chromium.disableJavascript = true;
          extraArgs = [ "--chromium-allow-list=file:///tmp/.*" ];
```

I think it is helpful to comment where we got this information from. Because apparently we are clueless about gotenberg and the paperless upstream isn't.

---

Good points, the link has now been added as a comment.